cloudformation/database/serverless-database.yaml

```diff
@@ -338,24 +338,36 @@ Resources:
 Outputs:
   ClusterName:
     Value: !Ref ClusterName
+    Export:
```

Comment:
Why is the target cluster trying to import the cluster name? Is this export assuming that the DB stack will be created before the Cluster?

Reply:
The assumption is definitely that the DB stack will be created before the cluster, and should operate independently of it. But, you are right that the customer does not explicitly _need_ the ClusterName to configure accounting on their ParallelCluster deployment. Shall I take it out?